### DISCUSSION TITLE
ch_PP-OCRv4_rec_infer模型转换onnx后推理正常，但是量化之后结果全错，可能是什么原因

### DISCUSSION THREAD

**Discussion by CodingPerNight** [PaddlePaddle/PaddleOCR#13579] [Q&A]
ch_PP-OCRv4_rec_infer模型转换onnx后推理正常，这是正常推理结果
![11111](https://github.com/user-attachments/assets/753aa671-3be0-4b88-8c59-36dcdad9dcd4)

进行int8量化之后，推理结果全是0
![2222222](https://github.com/user-attachments/assets/ba9efbe7-232a-421a-a06b-f37dafd52281)

量化脚本：
quantize_static(
    model_input=model_fp32,  # 输入模型
    model_output=model_quant_static,  # 输出模型
    calibration_data_reader=data_reader,  # 校准数据读取器
    quant_format=QuantFormat.QDQ,  # 量化格式 QDQ / QOperator
    activation_type=QuantType.QInt8,  # 激活类型 Int8 / UInt8
    weight_type=QuantType.QInt8,  # 参数类型 Int8 / UInt8
    calibrate_method=CalibrationMethod.MinMax,  # 数据校准方法 MinMax / Entropy / Percentile
)
请教一下，可能是什么原因

**Comment by SWHL** [comment]
这个倒是第一次遇到这个问题。平时用量化的版本少一些。
等等其他看到的小伙伴，看看能不能一起看看吧
